scripts/evaluate_ocr.py

```diff
@@ -128,31 +111,46 @@ def detect_top_languages(self, text, top_k=3):
     ]
 
     # Get line count for tqdm
-    n_lines = sum([1 for line in open_jsonl(in_jsonl, mode="rb")])
+    n_lines = sum(1 for line in xopen(in_jsonl, mode="rb"))
```

Comment:
yeah... it's a rabbit hole I've traveled several times myself. I believe that calling `xopen` _should_ be faster since we don't need to actualyl parse the json lines, just count the number of lines (and `xopen` is what the `orjsonl` package relies on (I checked their source code). So, we're not really introducing a new dependency, just now explicitly relying on an implicit one)